Automattic/pocket-casts-android
# Improving Automotive account details view

## Description
This change improves the Automotive account details section when you are both signed in and out.

This included:
- increasing the sizes of the small fonts
- increasing the size of the Plus badge
- increasing the profile picture size
- tidying up some of the constraint layouts so all the views have four anchors
- improving the appearance of the logout button
- improving the account created page

Fixes https://github.com/Automattic/pocket-casts-android/issues/966

## Testing Instructions
1. Deploy the Automotive build
2. Tap the settings cog
3. ✅  Verify the account details look correct
4. Login to your account
5. ✅  Verify the account details look correct
6. Tap on the account button 
7. ✅  Verify the account details look correct

## Screenshots 

| Before | After |
| --- | --- |
| ![Screenshot_20230517_235728](https://github.com/Automattic/pocket-casts-android/assets/308331/977b9a5e-cff3-413c-b531-e85ffabe8c55) | ![Screenshot_20230518_080630](https://github.com/Automattic/pocket-casts-android/assets/308331/260a6b24-8580-4b4a-9d7a-2f62582ea336) |
| ![Screenshot_20230517_235752](https://github.com/Automattic/pocket-casts-android/assets/308331/d5a4bc4d-4fe3-485b-96ef-19af084faaff) | ![automotive_signed_in_after](https://github.com/Automattic/pocket-casts-android/assets/308331/2f3fdbc3-f7e1-4272-9fd5-64c2d7689553) |
| ![Screenshot_20230517_235923](https://github.com/Automattic/pocket-casts-android/assets/308331/d011715d-b537-4c92-8131-e66bcf9222e9) | ![automotive_set_up_account_after](https://github.com/Automattic/pocket-casts-android/assets/308331/a3f9ccb9-c4a5-42ef-b3ac-edcd0dd067af) |
| ![Screenshot_20230518_080050](https://github.com/Automattic/pocket-casts-android/assets/308331/e856aee2-4473-41dd-adc7-a98e053cc729) | ![Screenshot_20230518_080452](https://github.com/Automattic/pocket-casts-android/assets/308331/838bd845-af0a-489d-be20-672561911a68) |
| ![mobile_before](https://github.com/Automattic/pocket-casts-android/assets/308331/0d0df7d8-9d21-43a3-8efd-74dd15bb79f4) | ![mobile_after](https://github.com/Automattic/pocket-casts-android/assets/308331/4b40f05c-50be-4dcb-811a-e017a18fd4fa) |
| ![mobile_before_accounts](https://github.com/Automattic/pocket-casts-android/assets/308331/6918a74f-4a8f-472a-b54a-bbf2c20eeec5) | ![Screenshot_20230518_090931](https://github.com/Automattic/pocket-casts-android/assets/308331/779eeb01-3302-4134-8ec0-669f560567b9) |
| ![Screenshot_20230518_093827](https://github.com/Automattic/pocket-casts-android/assets/308331/7439bec6-3d80-4f5a-936d-54a61fcbeaf1) | ![Screenshot_20230518_094418](https://github.com/Automattic/pocket-casts-android/assets/308331/a7a6fbc9-8035-42c2-99f8-ffb9c2a96d3d) |
| ![Screenshot_20230518_112710](https://github.com/Automattic/pocket-casts-android/assets/308331/01118461-cbf9-471a-8b44-07e58510b3ed) | ![Screenshot_20230518_120030](https://github.com/Automattic/pocket-casts-android/assets/308331/c551d80f-f22f-4a84-9ab0-0d85aaf42c9f) |
